src/proposals/20241017/SparkEthereum_20241017.sol

```diff
@@ -0,0 +1,160 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import { IERC20 } from "forge-std/interfaces/IERC20.sol";
+
+import { Ethereum, SparkPayloadEthereum, IEngine, Rates, EngineFlags } from 'src/SparkPayloadEthereum.sol';
+
+import { ICapAutomator } from "lib/sparklend-cap-automator/src/interfaces/ICapAutomator.sol";
+
+import { IMetaMorpho, MarketParams } from 'lib/metamorpho/src/interfaces/IMetaMorpho.sol';
+
+/**
+ * @title  Oct 17, 2024 Spark Ethereum Proposal
+ * @notice onboard sUSDS
+ *         Update Oracle for sDAI
+ *         Onboard Pendle sUSDe PTs to Morpho Spark DAI Vault
+ *         WBTC Changes
+ * @author Phoenix Labs
+ * Forum:  https://forum.sky.money/t/oct-3-2024-proposed-changes-to-spark-for-upcoming-spell/25293
+ * Vote:   https://vote.makerdao.com/polling/QmbHaA2G
+ *         https://vote.makerdao.com/polling/QmShWccA
+ *         https://vote.makerdao.com/polling/QmTksxrr
+ *         https://vote.makerdao.com/polling/QmSiQVWm
+ */
+contract SparkEthereum_20241017 is SparkPayloadEthereum {
+
+    address internal constant SUSDS            = 0xa3931d71877C0E7a3148CB7Eb4463524FEc27fbD;
+    address internal constant SUSDS_PRICE_FEED = 0x27f3A665c75aFdf43CfbF6B3A859B698f46ef656;
+
+    address internal constant SDAI_PRICE_FEED = 0x0c0864837C7e65458aCD3C665222203217019436;
```

Comment:
Sorry can you clarify what you mean here?

Reply:
it's a nitpick really, that it could be fully verified on etherscan so constructor args are listed at the bottom of the 'contract' tab and I don't have to go fish them out from the deployment transaction